### PR TITLE
Allow customizing bar height

### PR DESCRIPTION
**Pull request by davidyaha**
Hey @iRoachie!
Thanks for this awesome package!

I've needed to customize the barHeight so I went ahead and added it as prop.

Would it possible to merge and publish?

Thanks ahead!

### CODE REVIEW

File: src/lib/styles.js [hunk 1]
```diff
@@ -5,16 +5,21 @@ import values from './values';
...
 const Bar = styled.View`
   background-color: ${(props: BarProps) => props.barColor};
-  height: ${values.barHeight};
+  height: ${(props: BarProps) => props.barHeight || values.barHeight};
 `;
 
 const TabTrack = styled.View`
   flex-direction: row;
-  height: 46;
+  height: ${(props: BarProps) => (props.barHeight || values.barHeight) - 2};
```
Comment (iRoachie) [inline src/lib/styles.js]: It's better if you use defaultProps on MaterialTabs to set the default `props.barHeight` as `values.barHeight`. That way we'd avoid having to check this multiple times

**Comment by davidyaha** [comment]
@iRoachie For some reason jest fails to run. Something like this error - https://stackoverflow.com/questions/47781736/jest-cannot-find-module-setupdevtools-from-setup-js?rq=1
I'll try to figure it out but it may be quicker for you to update the snapshots if you don't mind?

Thanks!

File: src/lib/values.js [hunk 2]
```diff
@@ -1,5 +0,0 @@
-// @flow
-
-export default {
-  barHeight: 48,
```
Comment (iRoachie) [inline src/lib/values.js]: Don't delete this file, all the other places that need the height will get it from here.

File: src/components/MaterialTabs.js [hunk 3]
```diff
@@ -42,6 +43,7 @@ export default class MaterialTabs extends React.Component<Props, State> {
   static defaultProps = {
     selectedIndex: 0,
     barColor: '#13897b',
+    barHeight: 48,
```
Comment (iRoachie) [inline src/components/MaterialTabs.js]: import the values file here and get the value.

```js
import values from '../lib/values'

...

barHeight: values.barHeight
```

**Comment by iRoachie** [comment]
Alright just the snapshots now.

`npm run test — -u`

**Comment by davidyaha** [comment]
The jest problem I mentioned was happening since the 1st commit so it's not the flow issues I had on the second one unfortunately... 😞 
This is the output I get for some reason:
```
➜  react-native-material-tabs git:(master) yarn test -u
yarn run v1.3.2
warning ../../package.json: No license field
$ jest -u
 FAIL  src/__tests__/main.test.js
  ● Test suite failed to run

    Cannot find module 'setupDevtools' from 'setup.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:191:17)
      at Object.<anonymous> (node_modules/react-native/jest/setup.js:30:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.171s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by iRoachie** [comment]
Ah I see, could you make the other changes then?

**Comment by davidyaha** [comment]
I think the way I have it now will work best in the long run, the `barHeight` prop sets both the bar, tab and indicator height so all of those would correlate. let me know what you think of this commit here:
https://github.com/iRoachie/react-native-material-tabs/pull/16/commits/4d20e401e2541c700e68853cdc9c3186a9bb1762

**Comment by iRoachie** [comment]
Yea definitely will do that but I mean where we set the `barHeight` in the defaultProps. I'd prefer if you get the value from the values file instead of just writing `48`

**Comment by iRoachie** [comment]
Thanks for going through all the way!

**Comment by davidyaha** [comment]
Thanks @iRoachie! Again, awesome lib! saved me a bunch of time 👍

**Comment by iRoachie** [comment]
Live https://github.com/iRoachie/react-native-material-tabs/releases/tag/v3.3.0

**Pull request merged**